kafka/tests/test.sh

```diff
@@ -13,6 +13,13 @@ if [[ -z "${1:-}" ]]; then
 	exit 1
 fi
 
+tear_down() {
+  #Unload kafka and zookeeper
+  hab svc unload "${TEST_PKG_IDENT}" || true
+  hab svc unload core/zookeeper || true
+}
+trap tear_down EXIT
```

Comment:
@smacfarlane [Previously](https://github.com/habitat-sh/core-plans/pull/2829#discussion_r304436655) you mentioned not wanting a helper function for unload, does that extend to trap code?